src/PagePropsStatementCountLookup.php

```diff
@@ -14,13 +15,15 @@ public function __construct( LoadBalancer $loadBalancer ) {
 	}
 
 	public function getStatementCount( EntityId $entityId ) {
+		$wikibaseRepo = WikibaseRepo::getDefaultInstance();
+
 		$db = $this->loadBalancer->getConnection( DB_MASTER );
 
 		$res = $db->selectRow(
 			array( 'page_props', 'page' ),
 			array( 'pp_value' ),
 			array(
-				'page_namespace' => 0,
+				'page_namespace' => array_values( $wikibaseRepo->getEntityNamespaces() ),
```

Comment:
thanks for the pull request.

I think page_namespace here should be only the namespace for the given entity type (of the entityId) and not all entity namespaces.

we could get this from EntityNamespaceLookup.

also, EntityNamespaceLookup should be set in the constructor (ideally the dependency injected)